collectors/ebpf.plugin/README.md

```diff
@@ -237,6 +336,12 @@ receiving data for connections when the `UDP` protocol is used.
 Like the previous chart, this one also monitors `udp_sendmsg` and `udp_recvmsg`, but instead of showing the number of
 calls, it monitors the number of bytes sent and received.
 
+### Apps
+
+These are tracepoints related to OOM killing processes.
```

Comment:
Do we have/do we need any references to OOM Killer?

Reply:
If we do not add any information, this would be the unique chart without description. I think this will break a pattern we are bringing. I added an external link for users to have more details about these events.